{{ cookiecutter.project_slug }}/pyproject.toml

```diff
@@ -99,21 +100,19 @@ pydocstyle.convention = "google"
     "D100", # Missing docstring in public module
     "D104", # Missing docstring in public package
 ]
-{% if cookiecutter.packaging == "pip-tools" -%}
+
+
+{%- if cookiecutter.packaging == "pip-tools" or cookiecutter.packaging == "uv" %}
 [build-system]
 requires = ["setuptools>=78", "setuptools-scm"]
 build-backend = "setuptools.build_meta"
-{%- elif cookiecutter.packaging == "uv" -%}
+{%- else %}
 [build-system]
-requires = ["uv_build>=0.9.7,<0.10.0"]
-build-backend = "uv_build"
+requires = ["poetry-core>=2.0.0,<3.0.0"]
+build-backend = "poetry.core.masonry.api"
```

Comment:
Would the following look cleaner, and reduce the need for a longer `or` statement? (sorry the suggestion button wont work)
```
{%- if cookiecutter.packaging == "poetry" %}
[build-system]
requires = ["poetry-core>=2.0.0,<3.0.0"]
build-backend = "poetry.core.masonry.api"
{%- else %}
[build-system]
requires = ["setuptools>=78", "setuptools-scm"]
build-backend = "setuptools.build_meta"
```